packages/Popover/stories/examples/ScrollContainer.js

```diff
@@ -62,15 +65,52 @@ PopoverBox.propTypes = {
   getScrollContainer: PropTypes.func.isRequired,
 };
 
-const ExampleStory = props => (
-  <React.Fragment>
-    <CenteredStory>
-      <ScrollBox id="scroll-container">
-        <PopoverBox {...props} getScrollContainer={() => document.querySelector("#scroll-container")} />
-      </ScrollBox>
-    </CenteredStory>
-    <TrailerBox>Fin.</TrailerBox>
-  </React.Fragment>
-);
+function setKnobsTab() {
+  setTimeout(() => {
+    // eslint-disable-next-line no-restricted-globals
+    parent.document.querySelector("#storybook-panel-root .simplebar-content button").click();
+  });
+}
+
+function togglePanelPosition() {
```

Comment:
Kinda hacky, but this story will display better with the knobs at the bottom instead of on the right.

Reply:
🤔 hacky but smart

Reply:
There might be a way to do this with the new storybook component format.. but i can fix this later